tests/ListPool.UnitTests/ListPoolTests.cs

```diff
@@ -13,9 +14,10 @@ public class ListPoolTests
         public void Add_item_without_indicate_capacity_of_list()
         {
             int expectedItem = s_fixture.Create<int>();
-            using var sut = new ListPool<int>();
-
-            sut.Add(expectedItem);
+            using var sut = new ListPool<int>
```

Comment:
Why we should specify the scope of the using?

Reply:
just editor recommendation again, I can move it back :D